packages/twenty-server/src/modules/view/services/view.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import isEmpty from 'lodash.isempty';
+
+import { TwentyORMManager } from 'src/engine/twenty-orm/twenty-orm.manager';
+import { ViewFieldWorkspaceEntity } from 'src/modules/view/standard-objects/view-field.workspace-entity';
+
+@Injectable()
+export class ViewService {
+  private readonly logger = new Logger(ViewService.name);
+  constructor(private readonly twentyORMManager: TwentyORMManager) {}
+
+  async addFieldToViewsContainingOldField({
```

Comment:
IMO functions in services should be more generic. Input should be either:
- `addFieldToViews(workspaceId, newFieldId, viewFilter)`
-` addFieldToViews(workspaceId, newFieldId, views)`. Then you will need another function `findViews(workspaceId, filter)`